vue/components/ui/molecules/lightbox/lightbox.stories.js

```diff
@@ -11,13 +11,15 @@ storiesOf("Components/Molecules/Lightbox", module)
         },
         data: function() {
             return {
-                image: "https://sandbox.platforme.com/api/compose?brand=dummy&model=dummy"
+                image: "https://sandbox.platforme.com/api/brands/swear/logo.png?"

```

Comment:
Why are we swapping the default dummy for the story here

Reply:
rolled back to the original default